Resources/views/CRUD/base_show.html.twig

```diff
@@ -26,30 +26,32 @@ file that was distributed with this source code.
         {{ sonata_block_render_event('sonata.admin.show.top', { 'admin': admin, 'object': object }) }}
 
         {% for name, view_group in admin.showgroups %}
-            <div class="box box-primary">
```

Comment:
This class is obtained via ``{{ view_group.box_class }}``, see https://github.com/sonata-project/SonataAdminBundle/blob/master/Mapper/BaseGroupedMapper.php#L68.